recipes/rq/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "rq" %}
+{% set version = "0.6.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version}}.tar.gz
+  url: https://github.com/nvie/rq/archive/{{ version }}.tar.gz
+  sha256: 8c6c8b9328e4b0cb4e6abb11f9b75f44a80b91da07036a0079746302e759ef1c
+
+build:
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - rq = rq.cli:main
+    - rqinfo = rq.cli:info
+    - rqworker = rq.cli:worker
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - redis >=2.7.0
+    - click >=3.0
+
+  run:
+    - python
+    - redis >=2.7.0
+    - click >=3.0
+
+test:
+  imports:
+    - rq
+    - rq.cli
+    - rq.compat
+    - rq.contrib
+  commands:
+    - rq --help
+    - rqinfo --help
+    - rqworker --help
+
+about:
+  home: https://github.com/nvie/rq/
+  license: BSD
+  summary: 'RQ is a simple, lightweight, library for creating background jobs, and processing them.'
+
+extra:
+  recipe-maintainers:
+    - nvie
```

Comment:
@RSully Since I've no idea what this project is about, feel free to claim full maintainer ownership here and remove my name.